recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -0,0 +1,265 @@
+      cmds:
+        - |
+          # Check if a privileged container can be deployed in the namespace
+          cat << EOF | kubectl apply -n {{.NR_CLI_NAMESPACE}} -f -
+          apiVersion: v1
+          kind: Pod
+          metadata:
+            name: priv-check
+          spec:
+            hostNetwork: true
+            dnsPolicy: ClusterFirstWithHostNet
+            containers:
+            - name: check
+              image: newrelic/infrastructure-k8s
+              securityContext:
+                privileged: true
+              command: ["/bin/sh"]
+              args: ["-c", "sleep 60"]
+          EOF
+          if [[ $? -ne 0 ]]; then
+            echo "Failed to deploy a privileged container to the Kubernetes cluster."
+            echo "Turn on unprivileged mode and re-run the installation."
+            exit 31
+          fi
+        - |
+          kubectl delete pod priv-check -n {{.NR_CLI_NAMESPACE}}
+
+    download_manifest:
+      cmds:
+        - |
+          URL="https://k8s-config-generator.service.newrelic.com/generate?"
+          URL="${URL}global.cluster={{.NR_CLI_CLUSTERNAME}}"
+          URL="${URL}&global.namespace={{.NR_CLI_NAMESPACE}}"
+          URL="${URL}&newrelic-infrastructure.privileged={{.NR_CLI_PRIVILEGED}}"
+          URL="${URL}&global.lowDataMode={{.NR_CLI_LOW_DATA_MODE}}"
+          URL="${URL}&ksm.enabled={{.NR_CLI_KSM}}"
+          URL="${URL}&prometheus.enabled={{.NR_CLI_PROMETHEUS}}"
+          URL="${URL}&kubeEvents.enabled={{.NR_CLI_KUBE_EVENTS}}"
+          URL="${URL}&logging.enabled={{.NR_CLI_LOGGING}}"
+          URL="${URL}&global.licenseKey={{.NEW_RELIC_LICENSE_KEY}}"
+          if [ "{{.NEW_RELIC_REGION}}" = "STAGING" ]; then
+            URL="${URL}&global.nrStaging=true"
+          fi
+          if [ "{{.NR_CLI_NEWRELIC_PIXIE}}" = "true" ]; then
+            URL="${URL}&newrelic-pixie.enabled=true"
+            URL="${URL}&newrelic-pixie.apiKey={{.NR_CLI_PIXIE_API_KEY}}"
+            if [ "{{.NR_CLI_PIXIE}}" = "true" ]; then
+              URL="${URL}&pixie-chart.enabled=true"
+              URL="${URL}&pixie-chart.deployKey={{.NR_CLI_PIXIE_DEPLOY_KEY}}"
+              URL="${URL}&pixie-chart.clusterName={{.NR_CLI_CLUSTERNAME}}"
+              if [ ! -z "{{.NR_CLI_PIXIE_ADDRESS}}" ]; then
+                URL="${URL}&pixie-chart.cloudAddr={{.NR_CLI_PIXIE_ADDRESS}}"
+                URL="${URL}&pixie-chart.cloudUpdateAddr={{.NR_CLI_PIXIE_ADDRESS}}"
+              fi
+            fi
+          fi
+          curl -s "${URL}" > newrelic-k8s.yml
+
+    run_kubectl:
+      cmds:
+        - |
+          kubectl apply -f https://raw.githubusercontent.com/pixie-labs/pixie/main/k8s/operator/crd/base/px.dev_viziers.yaml
+        - |
+          kubectl apply -f https://raw.githubusercontent.com/pixie-labs/pixie/main/k8s/operator/helm/crds/olm_crd.yaml
+        - |
+          kubectl apply -f newrelic-k8s.yml
+
+    cleanup:
+      cmds:
+        - |
+          rm newrelic-k8s.yml
```

Comment:
See above

Reply:
I removed the rm and added a note in the postInstall.